# Position of SpeakEasy music playhead. Published about
# every 1/10th of a second.

float32  playhead_time  # time in fractional seconds from start of song.
